packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="okta-w-wrapper" ref="mainWrapper" >
+    <div id="widget-container" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'LiveWidgetOktaWidget',
+  props: ['configJS', 'configSCSS'],
+  data(){
+    return {
+      rendered: true
+    }
+  },
+  mounted: function() {
+   this.renderWidget()
+  },
+  watch: {
+    configJS: function(){
+      this.renderWidget()
+    },
+    configSCSS: function(){
+      //vue ignores empty style tags, so it's need to be done "old way"
+      if(this.configSCSS){
+        const styleTagToRemove = document.getElementById('styletag')
+        if (styleTagToRemove) { styleTagToRemove.remove()}
+        const style = document.createElement('style')
+        style.innerHTML = this.configSCSS
+        style.id = 'styletag'
+        this.$refs.mainWrapper.appendChild(style)
+      }
+    }
+  },
+  destroyed () {
+    this.widget ? this.widget.remove() : null;
+  },
+  methods:{
+    renderWidget(){
+      this.widget ? this.widget.remove() : null;
```

Comment:
Why use a ternary when
```js
if (this.widget) {
  this.widget.remove();
}```
Will suffice?

Also, is this worth refactoring into a `destroyWidget` function for reuse?